src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java

```diff
@@ -149,12 +157,20 @@ protected void setDefaults() {
         if (this.defaultProvisionedThroughput == null) {
             this.defaultProvisionedThroughput = 1L;
         }
+        if (this.billingMode == null) {
+            this.billingMode = BillingMode.PROVISIONED;
+        }
         if (streamsEnabled == null) {
```

Comment:
Don't we want the default to be `PAY_PER_REQUEST`

Reply:
I wasn't sure what default we wanted for mt-dynamo, since other clients like zos we will default to PAY_PER_REQUEST. I'll change the default in mt-dynamo to PAY_PER_REQUEST

Reply:
You're right.  I was thinking of the overall default.  At the mt-dynamo level, I suppose we should do whatever regular dynamo does.  That is, no default, it fails if you don't specify anything I think.